translations/nb.json

```
@@ -31,8 +31,20 @@
 	"PRECIP_POP": "Sannsynlighet for nedbør",
 	"PRECIP_AMOUNT": "Nedbørsmengde",
 
+	"MODULE_CONFIG_CHANGED": "Innstillingene for {MODULE_NAME} modulen har endret seg.\nVennligst sjekk dokumentasjonen.",
+	"MODULE_CONFIG_ERROR": "Feil i {MODULE_NAME} modulen. {ERROR}",
+	"MODULE_ERROR_MALFORMED_URL": "Ugyldig URL.",
+	"MODULE_ERROR_NO_CONNECTION": "Ingen internettforbindelse.",
+	"MODULE_ERROR_UNAUTHORIZED": "Autentisering mislyktes.",
+	"MODULE_ERROR_UNSPECIFIED": "Vennligst sjekk loggene for mer informasjon.",
+
+	"NEWSFEED_NO_ITEMS": "Ingen nyheter tilgjengelig nå.",
```

This is fine as it is, but we can improve the language a bit :blush:

```suggestion
	"MODULE_CONFIG_CHANGED": "Innstillingene for {MODULE_NAME}-modulen har endret seg.\nVennligst sjekk dokumentasjonen.",
	"MODULE_CONFIG_ERROR": "Feil i {MODULE_NAME}-modulen. {ERROR}",
	"MODULE_ERROR_MALFORMED_URL": "Ugyldig URL.",
	"MODULE_ERROR_NO_CONNECTION": "Ingen internettforbindelse.",
	"MODULE_ERROR_UNAUTHORIZED": "Autentisering mislyktes.",
	"MODULE_ERROR_UNSPECIFIED": "Vennligst sjekk loggene for mer informasjon.",

	"NEWSFEED_NO_ITEMS": "Ingen nyheter tilgjengelig for øyeblikket.",
```

---

Thank you @MagMar94! :smiley: